### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by sofimay**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
```
Comment (MrTim) [inline index.html:4]: Could you please put styles in separate file and connect it to index.html

File: index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        nav {
+            float: left;
+            width: 30%;
+            height: 400px;
+            background: #666;
+            padding: 20px;
+        }
+
+            /* Style the list inside the menu */
+            nav ul {
```
Comment (MrTim) [inline index.html:33]: pay attention to indentation. This block should be moved 4 spaces left to be aligned with the rest.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        @media (max-width: 1200px) {
+            nav, article {
+                width: 100%;
+                height: auto;
+            }
+        }
+    </style>
+</head>
+<body>
+
```
Comment (MrTim) [inline index.html:71]: too many empty lines

File: index.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+    <header>
+        <img src="img/logo2.svg" alt="logo" height="80px" width="80px"/>
+        <h1>Moon every day</h1>
+    </header>
+
+    <section>
+
```
Comment (MrTim) [inline index.html:79]: you don't need to create an empty line separator

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice start! I left some comments related to code.

**Comment by keponk** [comment]
@sofimay seems like this pull request has some conflicts. Normally this happens when git sees changes it doesn't know where it came from, like when we edit something in a different branch but not on this one.
 
Let us know if you need help completing this pull request :)

**Comment by sofimay** [comment]
> @sofimay seems like this pull request has some conflicts. Normally this happens when git sees changes it doesn't know where it came from, like when we edit something in a different branch but not on this one.
> 
> Let us know if you need help completing this pull request :)

I was wondering, if it won't be better to start a new pull request with all the changes, instead of merging this one? @keponk

**Comment by MrTim** [comment]
@sofimay I think it make sense to fix this pr and merge it. It could be a good practice in resolving the conflicts during merge, because this will happen from time to time.

**Pull request merged**